public/.htaccess

```diff
@@ -0,0 +1,6 @@
+RewriteEngine On
+RewriteBase /
```

Comment:
There's a reason this file wasn't already committed, which is that it is environment specific. This .htaccess would break environments that need to tweak the rewritebase, add additional options to force php7, etc.

Reply:
No problem, I can just provide a sample ht.access file that is copied into the project.